pkg/qemu/qemu.go

```diff
@@ -548,7 +548,7 @@ func Cmdline(ctx context.Context, cfg Config) (exe string, args []string, err er
 	// Machine
 	switch *y.Arch {
 	case limayaml.X8664:
-		if strings.HasPrefix(cpu, "qemu64") && runtime.GOOS != "windows" {
+		if accel == "tcg" && runtime.GOOS != "windows" { // FIXME: Any reason to exclude windows here?
```

Comment:
I think the check for "windows" was here to prevent the `accel == "whpx"` case to go in this branch, instead of the "else if" branch below.

Since you now test of `accel == "tcg"` instead of `cpu == "qemu64"`, I think you can remove the check for "windows" here.

Reply:
done